frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -32,6 +34,12 @@ export const billingLogic = kea<billingLogicType<BillingAlertType>>({
                     actions.registerInstrumentationProps()
                     return response as BillingType
                 },
+                setBillingLimit: async (billing: BillingType, breakpoint) => {
+                    await breakpoint(1000)
```

Comment:
Why?

Reply:
this was there, when I didn't have a submit button. Will remove.